Dockerfile

```
@@ -1,7 +1,8 @@
 # NOTE: this Dockerfile is purely for local development! it is *not* used for
 # the official 'concourse/concourse' image.
 
-FROM concourse/dev AS base
+ARG BASE_IMAGE=concourse/dev
+FROM ${BASE_IMAGE} AS base
```

Lowercase form is more conventional for build args; there are a few examples of this in their [docs](https://docs.docker.com/engine/reference/builder/#arg).

```suggestion
ARG base_image=concourse/dev
FROM ${base_image} AS base
```

---

Huh, weird. Elsewhere in the docs they use uppercase (https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact) (https://docs.docker.com/engine/reference/builder/#using-arg-variables)

Will change to lowercase, though - I like `IMAGE_ARG_base_image` over the alternative